src/database/index.ts

```diff
@@ -14,33 +14,83 @@
  * limitations under the License.
  */
 
-import { FirebaseApp } from '../firebase-app';
+import { app } from '../firebase-namespace-api';
 import { ServerValue as sv } from '@firebase/database';
-import * as adminDb from './database';
-import * as firebaseDbTypesApi from '@firebase/database-types';
-
-export declare function database(app?: FirebaseApp): adminDb.Database;
+import * as rtdb from '@firebase/database-types';
 
 /**
- * We must define a namespace to make the typings work correctly. Otherwise
- * `admin.database()` cannot be called like a function. Temporarily,
- * admin.database is used as the namespace name because we cannot barrel
- * re-export the contents from @firebase/database-types, and we want it to
- * match the namespacing in the re-export inside src/index.d.ts
+ * Gets the {@link admin.database.Database `Database`} service for the default
+ * app or a given app.
+ *
+ * `admin.database()` can be called with no arguments to access the default
+ * app's {@link admin.database.Database `Database`} service or as
+ * `admin.database(app)` to access the
+ * {@link admin.database.Database `Database`} service associated with a specific
+ * app.
+ *
+ * `admin.database` is also a namespace that can be used to access global
+ * constants and methods associated with the `Database` service.
+ *
+ * @example
+ * ```javascript
+ * // Get the Database service for the default app
+ * var defaultDatabase = admin.database();
+ * ```
+ *
+ * @example
+ * ```javascript
+ * // Get the Database service for a specific app
+ * var otherDatabase = admin.database(app);
+ * ```
+ *
+ * @param App whose `Database` service to
+ *   return. If not provided, the default `Database` service will be returned.
+ *
+ * @return The default `Database` service if no app
+ *   is provided or the `Database` service associated with the provided app.
  */
+export declare function database(app?: app.App): database.Database;
```

Comment:
What is it for?

Reply:
This is the signature of `admin.database()` API